packages/server/src/routes/provider/alert/create.svelte

```diff
@@ -546,36 +587,105 @@
         <button
           class="delete"
           aria-label="close"
-          on:click={() => (isIconSelectorOpen = false)} />
+          on:click={() => {
+            isIconSelectorOpen = false;
+            isCustomIconSelectorOpen = false;
+          }} />
       </header>
-      <section class="modal-card-body">
-        <div class="icon-choiced-container">
-          <div class="icon-choices">
-            {#each Object.values(icons) as icon}
-              <div class="icon-container">
-                <button
-                  class="button is-large"
-                  class:has-background-info={tempIcon === icon}
-                  on:click={() => (tempIcon = icon)}>
-                  <span class="icon is-large">
-                    <i class={`${icon} fa-2x`} />
-                  </span>
-                </button>
+      {#if isCustomIconSelectorOpen}
+        <section class="modal-card-body">
+          <div class="icon-choiced-container">
+            {#if iconSearchResults}
+              <div class="icon-choices">
+                {#each iconSearchResults as icon}
+                  <div class="icon-container">
+                    <button
+                      class="button is-large"
+                      class:has-background-info={tempIcon === icon}
+                      on:click={() => {
+                        tempIcon = icon;
+                        icons[icon] = icon;
+                      }}>
+                      <span class="icon is-large">
+                        <i class={`${icon} fa-2x`} />
+                      </span>
+                    </button>
+                  </div>
+                {/each}
+              </div>
+            {:else}
+              <div>
+                <h5>No results found</h5>
               </div>
```

Comment:
Instead of using an `h5` here, maybe we can spruce it up with [Bulma's notification component](https://bulma.io/documentation/elements/notification/). Minus the delete icon in their example, I think the `is-info` modification would work for this case.